README.md

```diff
@@ -32,9 +32,24 @@ or use the [com.strumenta.antlr-kotlin][1] plugin, which instructs ANTLR automat
 ## Kotlin Runtime
 
 The Kotlin runtime for the Kotlin target is derived from the Java runtime, and is built
-as a multi-platform project running on JVM, JS, and some of the Kotlin Native platforms.
-
-:warning: The Kotlin Native runtime is **not** thread safe
+as a multiplatform project running on JVM, JS, and Native.
+
+> :warning: The Native runtime is **not** thread safe
+
+The supported Native platforms are:
+
+| Tier 1            | Tier 2                | Tier 3   |
```

Comment:
As a user, it is relevant to know the difference in tiers?

Reply:
@ftomassetti probably not, but it's a way to keep them in sync with how they are displayed in the Kotlin Native Docs.
If you look at the Kotlin Docs, you can immediately map tiers to this table.

Reply:
Yes, but it also mean that if the list changes we need to update, so it is a tiny bit more work for us and no extra information for the user. I would rather link to the documentation and not duplicate the table

Reply:
We're not really duplicating the table tho. We're missing some targets from Tier 3, so if you match this table with the Kotlin Docs you immediately spot the difference.

Reply:
Oh, sorry, I see. Ok, then, merging